fix for: error: ‘struct hostent’ has no member named ‘h_addr’
author: Kenneth Hoste (HPC-UGent)
--- concorde/UTIL/safe_io.c.orig	2024-01-31 19:51:20.865401391 +0100
+++ concorde/UTIL/safe_io.c	2024-01-31 19:51:40.452228752 +0100
@@ -1248,7 +1248,7 @@
         fprintf (stderr, "cannot get host info for %s\n", hname);
         return (CC_SFILE *) NULL;
     }
-    memcpy ((void *) &hsock.sin_addr, (void *) h->h_addr, h->h_length);
+    memcpy ((void *) &hsock.sin_addr, (void *) h->h_addr_list[0], h->h_length);
     hsock.sin_family = AF_INET;
     hsock.sin_port = htons(p);
 
